examples/qml_extension_plugin/tests/main.cpp

```diff
@@ -0,0 +1,2 @@
+#include <QtQuickTest>
```

Comment:
missing license info

Reply:
Yup...
I really need to add a local configuration file to `cmps`, then I can finally have this automated :roll_eyes:

Reply:
Maybe I can do that in edu time :thinking: